airflow/config_templates/config.yml

```diff
@@ -672,7 +672,7 @@
       version_added: ~
       type: string
       example: ~
-      default: "Airflow HiveOperator task for {{hostname}}.{{dag_id}}.{{task_id}}.{{execution_date}}"
```

Comment:
Let me look at this one when I get to a computer, please

Reply:
I changed it slightly to handle both cases - when the option is missing and when it is empty. I do not really want to fix it now (it will not work if I add default because of single-curly-braces that are then resolved by Airflow's conf module using environment variables. 

I left TODO there and added an issue https://github.com/apache/airflow/issues/9005